src/c_api/c_api.cc

```diff
@@ -322,6 +322,38 @@ int MXNDArrayLoad(const char* fname,
   API_END();
 }
 
+int MXNDArrayLoadFileContent(const void *nd_file,
```

Comment:
It’s not really a file, right? More like LoadFromBuffer or something, right?

Reply:
changed to suggested name, which is better

Reply:
Was it changed in the c api header?

Reply:
Yup, for some reason this comment is linked to the old code. The header and c_api have been changed to MXNDArrayLoadFromBuffer